File: ver_cpp/T14.cpp
// https://leetcode.cn/problems/maximum-lcci/
#include<bits/stdc++.h>
using namespace std;

class Solution {
public:
    int maximum(int a, int b) {
        // return max(a,b);
        return a > b ? a : b;
    }
};

int main(){
    Solution s;
    int a,b;
    cin>>a>>b;
    cout<<s.maximum(a,b)<<endl;
    return 0;
}

File: ver_py/T14.py
# https://leetcode.cn/problems/maximum-lcci/
class Solution:
    def maximum(self, a: int, b: int) -> int:
        # return max(a, b)
        return a if a>b else b
    
if __name__ == "__main__":
    s = Solution()
    a, b = map(int, input().split())
    print(s.maximum(a, b))